pkg/deploy/generator/config_generator.go

```diff
@@ -60,12 +59,12 @@ func (g *DeploymentConfigGenerator) Generate(ctx kapi.Context, name string) (*de
 
 	refs, legacy := findReferences(dc)
 	if errs := retrieveReferences(g.Client, ctx, refs, legacy); len(errs) > 0 {
-		return nil, kerrors.NewAggregate(errs)
+		return nil, errors.NewInvalid("DeploymentConfig", name, errs)
 	}
 	indexed := referencesByIndex(refs, legacy)
 	changed, errs := replaceReferences(dc, indexed)
 	if len(errs) > 0 {
-		return nil, kerrors.NewAggregate(errs)
+		return nil, errors.NewInvalid("DeploymentConfig", name, errs)
```

Comment:
Kinds should be capitalized, the other spot is wrong and I'll fix it.  I'm only going to fix this file though.


> On Jan 27, 2015, at 1:48 PM, Dan Mace <notifications@github.com> wrote:
> 
> In pkg/deploy/generator/config_generator.go:
> 
> >  	}
> >  	indexed := referencesByIndex(refs, legacy)
> >  	changed, errs := replaceReferences(dc, indexed)
> >  	if len(errs) > 0 {
> > -		return nil, kerrors.NewAggregate(errs)
> > +		return nil, errors.NewInvalid("DeploymentConfig", name, errs)
> I meant just the actual naming/capitalization for a field/type reference.
> 
> —
> Reply to this email directly or view it on GitHub.
>